be/src/vec/exprs/vcompound_pred.h

```diff
@@ -61,7 +61,8 @@ class VCompoundPred : public VectorizedFnCall {
         int lhs_id = -1;
         int rhs_id = -1;
         RETURN_IF_ERROR(_children[0]->execute(context, block, &lhs_id));
-        ColumnPtr lhs_column = block->get_by_position(lhs_id).column;
+        ColumnPtr lhs_column =
+                block->get_by_position(lhs_id).column->convert_to_full_column_if_const();
```

Comment:
convert_to_full_column_if_const() was deleted by this pr

https://github.com/apache/doris/pull/26160/files